src/main/java/com/example/demo/models/UserModel.java

```diff
@@ -1,10 +1,19 @@
 package com.example.demo.models;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import javax.validation.constraints.Email;
+
 public class UserModel {
```

Comment:
let's not add validations to model, model should be as simple as possible, let's make UserRequest to handle receiving data from API and validating it